.ci/wercker.sh

```diff
@@ -244,10 +244,12 @@ no-error-spring-cloud-gcp)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   echo CS_version: ${CS_POM_VERSION}
-  checkout_from https://github.com/spring-cloud/spring-cloud-gcp
+  checkout_from https://github.com/romani/spring-cloud-gcp
   cd .ci-temp/spring-cloud-gcp
+  git checkout romani-patch-1
   mvn -e checkstyle:check@checkstyle-validation -P full-checkstyle \
-   -Dpuppycrawl-tools-checkstyle.version=${CS_POM_VERSION}
+   -Dpuppycrawl-tools-checkstyle.version=${CS_POM_VERSION} \
+   -Dmaven-checkstyle-plugin.version=3.1.1
```

Comment:
comment for disabling says
`disabled till project upgrade to new checkstyle maven plugin`
if we pass plugin version in cli, it means that it is not upgraded yet?

Reply:
As they upgrade to new their parent, we can remove this line probably. 
For now it is required.